.pre-commit-config.yaml

```diff
@@ -59,7 +59,7 @@ repos:
     hooks:
```

Comment:
```
  - repo: https://gitlab.com/pycqa/flake8
    rev: 3.9.2
```

in pyproject.toml we now use 4.0.1 so I'm trying that here, not sure why autoupdate didn't upgrade that though

Reply:
autoupdate won't catch the inner versions, these must be manually checked, e.g. https://pypi.org/project/flake8-eradicate/ is now at 1.2.1